zerver/models.py

```diff
@@ -4447,7 +4447,7 @@ class RealmAuditLog(AbstractRealmAuditLog):
         null=True,
         on_delete=CASCADE,
     )
-    event_last_message_id: Optional[int] = models.IntegerField(null=True)
+    event_last_message_id = models.IntegerField(null=True)
```

Comment:
Does this hunk belong in the previous commit?

Reply:
Fixed.

Reply:
No, that was actually intentional, as the previous commit doesn’t type check with this hunk.

Reply:
I see. `event_last_message_id = assert_is_not_none(log_entry.event_last_message_id)` fails.